content/reference/host-runtime/actors.md

```diff
@@ -0,0 +1,49 @@
+
+### Reactive
+
+Actors are [reactive](https://en.wikipedia.org/wiki/Reactive_programming). An actor cannot initiate any action on its own, it can simply _react_ to outside stimuli in the form of messages delivered by the host. Actor developers register message handlers that receive input and return messages as output, as shown in the following example that receives a bank account query and responds with the bank account value:
+
+```rust
+#[no_mangle]
+pub fn wapc_init() {
+    // other handlers...
+
+    bank::Handlers::register_handle_query(handle_balance_inquiry);
+}
+
+fn handle_balance_inquiry(query: bank::BalanceInquiry) -> HandlerResult<bank::Balance> {
+    let balance = // ... query
+    Ok(bank::Balance{ 
+        account: query.account,
+        balance
+    })
+}
+```
+
+While the preceding actor code could communicate with capability providers (e.g. a _key-value store_ to retrieve the account balance), it could only do so in response to a message being delivered from the host.
+
+### Communication by Abstraction
+
+wasmCloud actors are _loosely coupled_ with the capability providers they use for non-functional requirements. An actor doesn't communicate with **Redis** or **Cassandra** or **Consul**, instead it communicates with a generalized abstraction over _key-value stores_. Under the hood, each of these abstractions is represented by a contract or **interface**. As long as the capability provider implements the correct interface contract, it should be considered compatible with your actor. An actor written against the key-value store abstraction should be able to work with _any_ key-value store, and even have that store swapped live at runtime without requiring a rebuild or redeploy.
+
+There are a number of _first-party_ [actor interfaces](https://github.com/wasmcloud/actor-interfaces) which we maintain in our Github organization. Additionally, the community can build their own public capability providers and enterprises are free to build their own internal, proprietary capability providers that expose private enterprise or corporate functionality.
+
+### Secure
+
+Actors are secure by default. They must be explicitly granted access to each capability provider contract (abstraction/interface), otherwise the host runtime will not allow it to make calls to that provider or receive messages from it. Granting access to capabilities is discussed more in the [security](../security) section of the reference, but the short version is that each actor's WebAssembly module contains an embedded JSON Web Token (JWT) that holds claims, including claims attesting to which providers the actor is allowed to use.
```

Comment:
Should we include a small section here stating that, due to the structure of a webassembly module, Actors cannot perform these calls on their own? You've stated that actors are reactive and interact with providers, but we could add more clarity to say that Actors actually _cannot_ make that network request on its own